src/Microsoft.Identity.Web/TokenCacheProviders/Session/MsalSessionTokenCacheProvider.cs

```diff
@@ -41,6 +42,7 @@ public partial class MsalSessionTokenCacheProvider : MsalAbstractTokenCacheProvi
         public MsalSessionTokenCacheProvider(
             ISession session,
             ILogger<MsalSessionTokenCacheProvider> logger)
+            : base(null)
```

Comment:
An good exception should be thrown if this TokenCache impl is used and the Encyrpt flag is set to true.

Reply:
we don't use it for session cache.